blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineCreateRequest.java

```diff
@@ -60,8 +60,8 @@
     private static final Logger logger = LoggerFactory.getLogger(GithubPipelineCreateRequest.class);
 
     @DataBoundConstructor
-    public GithubPipelineCreateRequest(String name, String organization, BlueScmConfig scmConfig) {
-        super(name, organization, scmConfig);
+    public GithubPipelineCreateRequest(String name, BlueScmConfig scmConfig) {
```

Comment:
@michaelneale on closer look and some thingking, passing organization parameter is redundant - organization is always present to create pipeline, `POST /blue/rest/organizations/:orgId/pipelines/ -d {}`.